### ISSUE TITLE
Order of images in pdf is arbitrary

### ISSUE THREAD

**Issue by Nikita-Sherstnev** [navchandar/look-like-scanned#22]
**Describe the bug**
Order of images in generated pdf is arbitrary.

**To Reproduce**
Create a folder with enumerated images 1.jpg, 2.jpg ..., run `scanner -i .\folder -f "image"`

**Expected behavior**
Images in pdf are sorted by title.

**Device (please complete the following information):**
 - OS: Manjaro
 - Python Version 3.11
 - Package Version 1.0.1

**Comment by navchandar** [comment]
Thanks for creating the issue. TIL that the [file order is arbitrary](https://docs.python.org/3/library/os.html#os.listdir:~:text=The%20list%20is%20in%20arbitrary%20order).
So, I have updated the script to sort by name by default and provided more sort options.

Try updating to the latest version `pip install -U look-like-scanned` or `pip install look-like-scanned==1.0.2` and running again. I have tested this in Python Version 3.11 and MacOS though. Let me know if you are still facing issues.
